spec/integration/api_resources_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'swagger_helper'
+require 'rails_helper'
+
+describe ApiResourcesController do
+  let(:user) { create(:user) }
+
+  before do
+    sign_in user
+  end
+
+  path '/api/resources/{resource_id}' do
+    get 'Retrieves a resource' do
+      tags 'Resources'
+      produces 'application/json'
+      parameter name: :resource_id, in: :path, type: :string
```

Comment:
the resource_id parameter it's type integer, not string

Reply:
When it is sent within the url path, you receive it goes as a string

Reply:
I find it's more useful to declare the type of the parameter as declared in the schema, otherwise everything in a path is a string, which is maybe confusing. I was following this [example](https://github.com/ruslantolstov/rswag-example/blob/master/spec/requests/api/backoffice/articles_spec.rb) that followed that rule, but it's also true that it doesn't make any difference in the responses, neither the tests. In the official documentation, they treat id as string, so I think it's fine to declare it that way.